src/jit/liveness.cpp

```diff
@@ -1097,7 +1097,9 @@ void Compiler::fgExtendDbgLifetimes()
                 continue;
             }
 
-            // TODO(pdg): fix the code below for LIR!
+            // TODO-LIR: the code below does not work for blocks that contain LIR. As a result,
+            //           we must run liveness at least once before any LIR is created in order
+            //           to ensure that this code doesn't attempt to insert HIR into LIR blocks.
 
```

Comment:
Is the "TODO" to make the code work? Will we actually do that? Or, will we just live with the status quo? If the latter, the comment can stay, but remove the "TODO-LIR" tag.